frontend/app/model/Eventbrite.scala

```diff
@@ -187,6 +187,7 @@ object Eventbrite {
       case t: InternalTicketing => t
     }
 
+    val ticketsNotSold = capacity - ticket_classes.map(_.quantity_sold).sum
```

Comment:
Might still need the `ticketsNotSold` if you do want to display the count in the view though.